docs/COMPILATION.MD

```diff
@@ -6,7 +6,7 @@ Downloads for precompiled binaries and source code can be found [on our website]
 
 Clone the latest repository from Github
 
-```
+```bash
 git clone https://github.com/CCExtractor/ccextractor.git
```

Comment:
Why do we want that if we're already in a terminal? :-)

Reply:
All the following code blocks also have the that so I just added it for consistency.